project/src/test/java/com/google/sps/SearchStoreTest.java

```diff
@@ -92,166 +94,250 @@
               new Keyword("Event", 0.06f),
               new Keyword("closure", 0.04f),
               new Keyword("attendees", 0.03f)));
-  private static final String TITLE_WITH_GAMES_IN_HIGH_RELEVANCE =  "End of the Year Games and Picnic";
-  private static final ArrayList<Keyword> KEYWORDS_TITLE_WITH_GAMES_IN_HIGH_RELEVANCE =
+  private static final String NAME_WITH_GAMES_IN_HIGH_RELEVANCE =  "End of the Year Games and Picnic";
+  private static final ArrayList<Keyword> KEYWORDS_NAME_WITH_GAMES_IN_HIGH_RELEVANCE =
       new ArrayList<Keyword>(
           Arrays.asList(new Keyword("picnic", 0.56f), new Keyword(GAMES, 0.17f)));
 
   @Before
   public void setUp() throws Exception {
+    // Mock a request to trigger the SpannerClient setup to run.
+    MockServletContext mockServletContext = new MockServletContext();
+    new SpannerClient().contextInitialized(new ServletContextEvent(mockServletContext));
+    SpannerTestTasks.setup();
+
     mockKeywordHelper = Mockito.mock(KeywordHelper.class);
     searchStore = new SearchStore(mockKeywordHelper);
   }
 
+  @After
+  public void tearDown() throws Exception {
+    SpannerTestTasks.cleanup();
+  }
+
   @Test
-  public void oneEvent_KeywordNotRelevantInEventTitleOrDescription_noResultsReturned()
+  public void oneEvent_KeywordNotRelevantInEventNameOrDescription_noResultsReturned()
       throws IOException {
-    // ID         |   Title Has Games  |   Description Has Games
-    // 1                    No                  No    
-
-    // On the first call the getKeywords for title, return empty list
+    // ID         |   Name Has Games  |   Description Has Games
+    // 1                    No                  No
+    SpannerTasks.insertorUpdateEvent(
+        TestUtils.newEventWithFutureDate(EVENT_ID_1, NAME_WITHOUT_GAMES, DESCRIPTION_WITHOUT_GAMES));
+    // On the first call the getKeywords for name, return empty list
     // On the second call the getKeyords for description, return empty list
     Mockito.when(mockKeywordHelper.getKeywords())
-        .thenReturn(KEYWORDS_TITLE_WITHOUT_GAMES, KEYWORDS_DESCRIPTION_WITHOUT_GAMES);
+        .thenReturn(KEYWORDS_NAME_WITHOUT_GAMES, KEYWORDS_DESCRIPTION_WITHOUT_GAMES);
     
-    searchStore.addEventToIndex(EVENT_ID_1, TITLE_WITHOUT_GAMES, DESCRIPTION_WITHOUT_GAMES);
-    List<String> actualResults = searchStore.getSearchResults(GAMES);
+    searchStore.addEventToIndex(EVENT_ID_1, NAME_WITHOUT_GAMES, DESCRIPTION_WITHOUT_GAMES);
+    List<Event> actualResults = searchStore.getSearchResults(GAMES);
```

Comment:
Usually we don't call 2 functions in one test because the test can pass when none of these functions work. For example, addEventToIndex doesn't work properly so nothing is added, getSearchResults doesn't work properly but since there is nothing in the DB, so the test passes.

I understand that you write it this way because you have no other way to add Event to your DB. So I would recommend you to add a comment for the test to describe it tests 2 behaviors. And also Assert if the Event is added to DB before  getSearchResults